src/amo/actions/reviews.js

```diff
@@ -611,6 +611,7 @@ export const hideFlashedReviewMessage = (): HideFlashedReviewMessageAction => {
 
 type DeleteAddonReviewParams = {|
   errorHandlerId: string,
+  isReplyToReviewId?: number,
```

Comment:
nit: I guess when I see a var start with "is" like this I think it's a boolean. can this just be replyToReviewId ? or is that more involved/ a bigger patch for a change like this?

Reply:
Yeah, I thought the same thing at first, but this is what the same variable is called elsewhere. I suppose I can just change it here.

Reply:
yeh I kind of remember that (it's in other places)..I think this is a nit so i would say only if it's not too involved otherwise that can be a follow up patch/issue if you think it's worth it..